### PR TITLE
Updates by vezza

### PR DESCRIPTION
**Pull request by legoboyvdlp**
-

### CODE REVIEW

**Comment by vezza** [comment]
- wheels smoke
- wet wheels spray
- brakes temp (to be continued, missing ECAM messages)
- bugfix (A320-100 fuel system, please check)

**Comment by legoboyvdlp** [comment]
Merely landing with MED braking caused brake smoke, it probably needs tuned? And surely it should be adjusted to actual braking decleration, not a fixed value.

**Comment by legoboyvdlp** [comment]
Everything appears to be fine, if you could tune it so it doesn't smoke unless you brake more severly would be god other than that I will merge once you do that

**Comment by legoboyvdlp** [comment]
Pushed my corrections / re-organization of file to vezza-master branch.

**Comment by vezza** [comment]
Yes it needs to be tuned... I'm not an expert so it will take me some time to work out how to set some good values... also I'd like to set 2 levels of overheating, the first with only smoke coming from the main landing gear, the second with also the firetrucks

**Comment by vezza** [comment]
I think now it should be acceptable... can you please test it?

**Comment by legoboyvdlp** [comment]
Thanks will do.

File: A320-100-CFM-set.xml [hunk 1]
```diff
@@ -163,6 +163,14 @@
 			</config>
 		</view>
 		
+		<systems>
+      <wingflexer include="Systems/wingflexer-params.xml"/>
```
Comment (legoboyvdlp) [inline A320-100-CFM-set.xml]: Formatting. Please use tabs and make sure its consistently indented.

File: A320-200-CFM-set.xml [hunk 2]
```diff
@@ -163,6 +163,14 @@
 			</config>
 		</view>
 		
+		<systems>
+      <wingflexer include="Systems/wingflexer-params.xml"/>
```
Comment (legoboyvdlp) [inline A320-200-CFM-set.xml]: same again

**Comment by legoboyvdlp** [comment]
Actually, I'll fix all this stuff but anyway, be aware of the formatting esp, please.

File: A320-100-CFM-set.xml (hunk 1 above)
Comment (legoboyvdlp) [reply A320-100-CFM-set.xml]: Besides, its a duplicate of what is below and also can go in -main, anyway.

**Comment by legoboyvdlp** [comment]
Thank you.

**Comment by vezza** [comment]
Ok, sorry... I'm quite a noob with git and xml, I'm learning by doing... thanks

**Comment by vezza** [comment]
I've fixed my master branch as you suggested...

**Pull request merged**